Sources/CoreMetrics/Metrics.swift

```diff
@@ -145,6 +149,10 @@ public class Recorder {
 
     /// Record a value.
     ///
+    /// Recording a value is meant to have "set" semantics, rather than "add" semantics.
+    /// Metrics implementations may choose to provide additional "add" or other operations meant to modify an existing value,
```

Comment:
this seems out of place in the user facing API, maybe move to the Hander API which is aimed for the backend dev?

Reply:
I think it somewhat makes sense to have this here, for example with Prometheus, if you needed/wanted `add` semantics, you would know that 1) `.record()` has `set` semantics and that 2) you could look into if the underlying `PrometheusClient` does have an `add` operation (which in the case of Prometheus, it does)

Reply:
while i fully recognize the value in using the potentially richer api the "backend" offers, the design goal of metrics-api (and log-api fwiw) is that end users wont need to worry about the underlying "backend". using the "backend" api  is not even an option for libraries that integrate with these apis as they are not exposed to the "backend" which is dictated by the application, but the idea is also true for applications in vast majority of the cases. as such, i think we should not suggest switching to the "backend" api here as it send the opposite message. how about we add a section in the readme that explains this? /cc @ktoso @mrlotu

Reply:
@ktoso @MrLotU wdyt?

Reply:
Whoops seems I missed some emails last week here; Yes, agreed @tomerd 
The API here should not go around and make advertisements for what libraries can't use anyway 👍 

Amending now